### DISCUSSION TITLE
Information on Sleep and Deep Sleep modes in PolarFire SoC

### DISCUSSION THREAD

**Discussion by ManishaI31214** [polarfire-soc/polarfire-soc-documentation#128] [Reference designs, FPGA and development kits] [answered]
If there are unused cores. How can they be put it in sleep mode?
In the the PFSoC register map we came across "RAM_SHUTDOWN_CR (0x2000 2170)" & "RAM_DEEPSLEEP_CR(0x2000 216C)" registers.
I think we don’t have an option to put the PolarFire SoC cores in sleep or deep sleep mode. The PolarFire SoC cores can only be kept in WFI mode. The registers RAM_SHUTDOWN_CR, RAM_DEEPSLEEP_CR are used to put the RAM blocks in the core in sleep mode but not the complete core, am I correct, please confirm

**Comment by hughbreslin** [comment, marked as answer]
Hi @ManishaI31214 

correct there is no way to gate the clock to a hart and put it into a "sleep" mode. The registers you mentioned only affect the ram blocks themselves
